### DISCUSSION TITLE
Feature Request: Customizable Split Ratio for Tiles Layout

### DISCUSSION THREAD

**Discussion by nohzafk** [nikitabobko/AeroSpace#639] [feature-ideas]
Thanks for developing such a great app! I wanted to suggest an enhancement regarding the tiles layout. In the current configuration, all windows are split equally to a ratio of 0.5. I am wondering if it's possible to introduce a setting that allows users to specify their desired split ratio.

Personally, I would prefer a golden ratio of 0.618. This is slightly larger than 0.5 and would help to ensure that my eyes are not constantly drawn to the split in the middle of the windows. At present, to avoid this distraction, I need to sit on one side of the table or constantly adjust my focus towards one side of the monitor.

This adjustment could work similarly to the split ratio settings in yabai, which you can read more about here: [yabai split ratios](https://github.com/koekeishiya/yabai/wiki/Configuration#split-ratios).

Thank you!

**Comment by idvorkin** [comment]
Has anyone done a workaround for this? I was thinking some kind of script that would look at all the window sizes and then do a manual size command?

When I tried to play with this, I didn't find a command to find the current window sizes, any insights/scripts would be much appreciated!

**Comment by CheariX** [comment]
Yes. This is something that I'm really missing.

Currently, aerospace only knows about absolute values.
E.g., if you have to windows and increase one window's size so that you have your ration (say 2:1), then it only works on this monitor.
If i swap the workspace to another monitor (e.g, internal LCD), the ration looks totally different due to the absolute values do note represent a usefull ration on this screen.

I'd really love to have this feature!

**Comment by fabrizioanichini** [comment]
I'd also love this feature. My use case is maintaining golden ratio proportions (0.618) for aesthetic harmony across workspaces. Currently using AeroSpace for workspace management and would prefer not to switch to yabai just for this feature.

  A simple config like:
  ```toml
  split-ratio = 0.618
```
  would be ideal.

**Comment by BarutSRB** [comment]
[https://github.com/BarutSRB/HyprSpace/commit/a37ec2081d4f15aedf0ef7c8c2f2f614a1ffca39](https://github.com/BarutSRB/HyprSpace/commit/a37ec2081d4f15aedf0ef7c8c2f2f614a1ffca39)
